Chummer/frmCareer.cs

```diff
@@ -16313,6 +16368,295 @@ private void AddCyberwareSuite(Improvement.ImprovementSource objSource)
 
             IsDirty = true;
         }
+
+        /// <summary>
+        /// Refreshes the Sustained Spells Panel
+        /// </summary>
+        /// <param name="panSustainedSpells"></param>
+        /// <param name="notifyCollectionChangedEventArgs"></param>
+        private void RefreshSustainedSpells(Panel flpSustainedSpells, Panel flpSustainedComplexForms, Panel flpSustainedCritterPowers, NotifyCollectionChangedEventArgs notifyCollectionChangedEventArgs = null)
+        {
+            if (flpSustainedSpells == null && flpSustainedComplexForms == null && flpSustainedCritterPowers == null)
+                return;
+
+            if (CharacterObject.SustainedCollection.Count != 0 && !chkPsycheActive.Visible)
+            {
+                //Sets up the Psyche Active Checkbox
+                chkPsycheActive.Visible = true;
```

Comment:
Setting this checkbox visible here and then not-visible later will cause flickering. You should find a different way of updating this checkbox's visibility.